src/PlatformSwitch/PlatformSwitch.tsx

```diff
@@ -19,6 +20,10 @@ export type PlatformSwitchProps = {
   accentColor?: string;
   /** Color of the icons when the corresponding platform is disabled */
   disabledColor?: string;
+  /** Tooltips to add to the buttons, in order left to right */
+  tooltips?: string[];
+  /** Placements of tooltips, in order left to right */
+  placements?: TooltipProps['placement'][];
```

Comment:
To avoid passing indexed arrays, have a look at `platformProps` at line 32 below: it already provides you with a mapping from platform to props. Consider converting from arrays to a platform prop key here (simply add `tooltip?: string` and `placement?: TooltipProps['placement']`)

https://github.com/graasp/graasp-ui/blob/70cdc2916da4d38ff423a685e3aef0923e2d5b03/src/PlatformSwitch/PlatformSwitch.tsx#L32-L51